common/src/main/java/io/homeassistant/companion/android/common/data/keychain/KeyChainRepositoryImpl.kt

```diff
@@ -0,0 +1,62 @@
+package io.homeassistant.companion.android.common.data.keychain
+
+import android.content.Context
+import android.security.KeyChain
+import io.homeassistant.companion.android.common.data.prefs.PrefsRepository
+import java.security.PrivateKey
+import java.security.cert.X509Certificate
+import java.util.concurrent.Executors
+import javax.inject.Inject
+
+class KeyChainRepositoryImpl@Inject constructor() : KeyChainRepository {
+
+    @Inject
+    lateinit var prefsRepository: PrefsRepository
+
+    private var alias: String? = null
+    private var key: PrivateKey? = null
+    private var chain: Array<X509Certificate>? = null
+    private var isLoading: Boolean = false
+
+    override suspend fun getAlias(): String? {
+        return alias
+    }
```

Comment:
The `suspend` modifier could be removed from this method as well.

Reply:
Done